<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayListGUI</class>
 <widget class="QFrame" name="PlayListGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlayListGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
	border-width: 5px;
	border-color: grey;
	border-style: inset;
	border-radius: 20px;
	background: url(:/images/channels.png);
}

QPushButton
{
	color: white;
	border: 1px solid black;
	border-radius: 5px;
	background-color: QLinearGradient(x1:0, y1:0,
															 x2:0, y2:1,
															 stop:0 #616161,
															 stop: 0.5 #505050,
															 stop: 0.6 #434343,
															 stop:1 #656565);
	height: 22px;
	font: bold 13px;
}

QPushButton:pressed
{
	background-color: QLinearGradient(x1:0, y1:0,
															 x2:0, y2:1,
															 stop:0 #414141,
															 stop: 0.5 #303030,
															 stop: 0.6 #232323,
															 stop:1 #454545);
}

QPushButton:checked
{
	background-color: QLinearGradient(x1:0, y1:0,
															 x2:0, y2:1,
															 stop:0 #b1b100,
															 stop: 0.5 #a0a000,
															 stop: 0.6 #939300,
															 stop:1 #b5b500);
}

QPushButton:disabled
{
	background-color: transparent;
	color: transparent;
	border: 0px;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="pushButtonAddFile">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Add File</string>
   </property>
  </widget>
  <widget class="PlayListTable" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QAbstractItemView
{
    background-color: QLinearGradient(x1: 0, y1: 0,
															 x2: 0, y2: 1,
															 stop: 0 #4d4d4d,
															 stop: 0.1 #646464,
															 stop: 1 #5d5d5d);
	border: none;
	background: rgb(70, 70, 70);
}

QTableWidget
{
	border: 2px solid darkgray;
	border-radius: 0px;
	background: rgb(70, 70, 70);
    selection-background-color: QLinearGradient(x1: 0, y1: 0,
																		x2: 0, y2: 1,
																		stop: 0 #ffa02f,
																		stop: 1 #d7801a);
}

QHeaderView::section
{
	color: white;
    border: 1px solid #6c6c6c;
	height: 30px;
	font: bold 20px;
	padding-bottom: 5px;
    background-color: QLinearGradient(x1:0, y1:0,
															 x2:0, y2:1,
															 stop:0 #616161,
															 stop: 0.5 #505050,
															 stop: 0.6 #434343,
															 stop:1 #656565);
}

QScrollBar:vertical
{
      background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 0.0 #121212, stop: 0.2 #282828, stop: 1 #484848);
      width: 5px;
      margin: 16px 0 16px 0;
      border: 1px solid #222222;
}

QScrollBar::handle:vertical
{
      background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 0.5 #d7801a, stop: 1 #ffa02f);
      min-height: 20px;
      border-radius: 2px;
}

QScrollBar::add-line:vertical
{
      border: 1px solid #1b1b19;
      border-radius: 2px;
      background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
      height: 14px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical
{
      border: 1px solid #1b1b19;
      border-radius: 2px;
      background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #d7801a, stop: 1 #ffa02f);
      height: 14px;
      subcontrol-position: top;
      subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
{
      border: 1px solid black;
      width: 1px;
      height: 1px;
      background: white;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
      background: none;
}

QAbstractScrollArea::corner
{
      background: darkGray;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="pushButtonRemoveFile">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChain">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Chain</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRecuePlaylist">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Recue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Loop</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBlock">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Block</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>pushButtonLoop</zorder>
  <zorder>pushButtonAddFile</zorder>
  <zorder>tableWidget</zorder>
  <zorder>pushButtonRemoveFile</zorder>
  <zorder>pushButtonChain</zorder>
  <zorder>pushButtonRecuePlaylist</zorder>
  <zorder>pushButtonBlock</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlayListTable</class>
   <extends>QTableWidget</extends>
   <header>playlisttable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
